docs/topics/mta-rn-resolved-issues-6-2-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-6.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-6-2-0_{context}"]
+= Resolved issues
+
+The following issues have been resolved in MTA version 6.2.0.
```

Comment:
Looking at the format use in [existing documentation](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.1/html/release_notes/mta-6-1-0#mta-rn-resolved-issues-610_release-notes) please can you use this format for now? 

I honestly prefer the Jira filter than the list